src/App.js

```diff
@@ -25,9 +35,11 @@ function App() {
   }
 
   const HandleKeyDown = (event) => {
-    switch (event.key.toLowerCase()){
+    switch (event.key.toLowerCase()) {
       case "s":
-        getStartTimeStamp()
+        if(recordState!==true){
```

Comment:
lets get some consistent formatting going. Let's add a formatter to the project soon. I'd suggest prettier:

https://prettier.io/

Reply:
Added prettier